frontend/src/components/AboutMe/ProfileCard.jsx

```
@@ -99,21 +99,29 @@ const ProfileCard = () => {
-          </div>
+        {/* Hover Overlay - only shows over the image */}
+        <div
+          className="absolute inset-0 rounded-full bg-black bg-opacity-50 opacity-0 group-hover:opacity-100 flex items-center justify-center transition duration-300 cursor-pointer"
+          onClick={triggerFileInput}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter' || e.key === ' ') {
+              triggerFileInput();
+            }
+          }}
+          tabIndex={0} // Makes the div focusable for keyboard navigation
+          role="button" // Specifies this is a button-like element
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)